### PR TITLE
Add a few more helpful link in the README

### PR DESCRIPTION
**Pull request by nickrobinson251**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,27 +5,31 @@
...
+```
 
 ## Usage
 
 ### Interactive Generation
 
 You can fully customize your package interactively with:
 
-```julia
+```jl
```
Comment (oxinabox) [inline README.md:30]: what is the difference?

**Review by oxinabox — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -65,9 +69,10 @@ t = Template(;
...
 
-For a much more detailled overview, please see the documentation.
+For a much more detailed overview, please see [the User Guide documentation](https://invenia.github.io/PkgTemplates.jl/stable/user/).
 
 ## Contributing
 
 Issues and pull requests are welcome!
-For some more specific tips, see the developer documentation.
+New contributors should make sure to read the [ColPrac Contributor Guide](https://github.com/SciML/ColPrac).
+For some more PkgTemplates-specific tips, see the [Developer Guide documentation](https://invenia.github.io/PkgTemplates.jl/stable/developer/).
```
Comment (oxinabox) [inline README.md:78]: colprac should also be referenced in that docs page

File: README.md (hunk 1 above)
Comment (nickrobinson251) [reply README.md:30]: none, we jsut use `jl` everywhere else

File: README.md (hunk 2 above)
Comment (nickrobinson251) [reply README.md:78]: okay :) added

**Pull request merged**